modules/ROOT/pages/release-notes.adoc

```diff
@@ -10,8 +10,8 @@ The 2022.2 is in Beta.
 
 === Bonita now embeds the bpmn-visualization library to improve process monitoring for administrators and users 
```

Comment:
This title is too long, see as it appears in the toc at the left of the page 

![image](https://user-images.githubusercontent.com/27200110/183646582-5efc65e6-447e-4df4-b1ae-27415fd721a7.png)

Reply:
done in e3dab0a